packages/rancher-cis-benchmark/charts/templates/view-role.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  namespace: {{ template "cis.namespace" . }}
+  name: cis-view
+rules:
+  - apiGroups:
+      - cis.cattle.io
+    resources:
+      - clusterscanbenchmarks
+      - clusterscanprofiles
+      - clusterscans
+      - clusterscanreports
+    verbs: ["get", "watch", "list"]
```

Comment:
nit: its nice to have the admin/edit/view all in the same file, that way if you have to change a resource down the line you don't forget about one.

Reply:
This looks good now but can you re-name the file from `admin-roles` to just `roles` or something generic.

Reply:
done - renamed as cis-roles.yaml